api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/WeightLoader.kt

```diff
@@ -255,7 +252,7 @@ private fun fillLayerVariablesFromKeras(layerName: String,
         require(variable.shape.toIntArray().contentEquals(dims)) {
             "${variable.name} variable shape in loaded data is ${dims.contentToString()}. Should be ${variable.shape.toIntArray().contentToString()}"
         }
-        model.fillVariable(variable.name, it.data)
```

Comment:
I'm in doubts regarding changing model.fillVariable to variable.fill by model. Session is related to model and initialized variables are too belong to model. Looks like changing of the sense and accents.

Reply:
Moved `init` and `fill` methods to the model.